source/common/config/cds_json.cc

```diff
@@ -97,6 +97,12 @@ void CdsJson::translateOutlierDetection(
   JSON_UTIL_SET_INTEGER(json_outlier_detection, outlier_detection, success_rate_minimum_hosts);
   JSON_UTIL_SET_INTEGER(json_outlier_detection, outlier_detection, success_rate_request_volume);
   JSON_UTIL_SET_INTEGER(json_outlier_detection, outlier_detection, success_rate_stdev_factor);
+  JSON_UTIL_SET_INTEGER(json_outlier_detection, outlier_detection,
```

Comment:
Please revert this file, we should no longer be supporting any v1 config changes.